assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -91,11 +91,8 @@ export const MESSAGES = {
     COVER_WRONG_ASPECT_RATIO: {
       MAIN_TEXT: __('Story cover image wrong aspect ratio', 'web-stories'),
       HELPER_TEXT: sprintf(
-        /* translators: 1: minimum cover dimension width X minimum cover dimension height. 2: cover dimensions aspect ratio. */
-        __(
-          'Should be at least %1$s and maintain a %2$s aspect ratio.',
-          'web-stories'
-        ),
+        /* translators: 1: cover dimensions aspect ratio. */
+        __('Should maintain a %2$s aspect ratio.', 'web-stories'),
         `${COVER_DIMENSION_WIDTH_PX}x${COVER_DIMENSION_HEIGHT_PX}px`,
         `${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`
```

Comment:
where's `%1$s`? 😄 

If there is only one placeholder, simply use `%s`. If there are multiple, number them starting with index 1, not 2.

```js
/* translators: %s: image aspect ratio. */
__('Should maintain a %s aspect ratio.', 'web-stories'),
`${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`

Reply:
Ah good find.